src/Revenue/Listeners/DeleteRevenueWhenDonationDeleted.php

```diff
@@ -26,11 +26,12 @@ public function __construct( Revenue $revenueRepository ) {
 	 *
 	 * @since 2.9.2
```

Comment:
@ravinderk Please add `@since 2.9.4 removed $post parameter for < WP 5.5 compatibility`

Reply:
fixed: https://github.com/impress-org/givewp/compare/e29d0e05663a...8f1ad95919eb